keyboards/preonic/keymaps/arkag/rules.mk

```
@@ -0,0 +1,20 @@
+BOOTMAGIC_ENABLE = lite
+MOUSEKEY_ENABLE = no
+STENO_ENABLE = no
+EXTRAKEY_ENABLE = yes
+CONSOLE_ENABLE = no
+COMMAND_ENABLE = no
+SLEEP_LED_ENABLE = yes
+NKRO_ENABLE = yes
+BACKLIGHT_ENABLE = no
+AUDIO_ENABLE = yes
+RGBLIGHT_ENABLE = yes
+TAP_DANCE_ENABLE = no
+UNICODE_ENABLE = yes
+LEADER_ENABLE = yes
+
+# save me space!
+EXTRAFLAGS += -flto
```

This does the same, plus some additional stuff. 
```suggestion
LTO_ENABLE = yes
```